src/dune_load.ml

```diff
@@ -265,7 +265,7 @@ let interpret ~dir ~project ~ignore_promoted_rules
 let load ?(ignore_promoted_rules=false) ~ancestor_vcs () =
   let ftree = File_tree.load Path.Source.root ~ancestor_vcs in
   let projects =
-    File_tree.fold ftree ~traverse_ignored_dirs:false ~init:[]
+    File_tree.fold ftree ~traverse_data_only_dirs:false ~traverse_vendored_dirs:true ~init:[]
```

Comment:
Rather than pass these long command line arguments (which are going way over the 80 char margin), perhaps we can have a set argument:

```
module Status : sig
  type t = Data_only | Vendored | Normal
  module Set : sig
    type status
    type t =
      { data_only : bool
      ; vendored : bool
      ; normal : bool
      }
   val mem : t -> status -> bool
  end with type status := t
end

val fold : .. -> traverse:Status.Set.t -> ..
```

Reply:
Sounds good, I'll change that!